src/client/src/shell/routes/BlotterRoute.tsx

```diff
@@ -5,9 +5,9 @@ import { styled } from 'rt-theme'
 
 const BlotterContainerStyle = styled('div')`
   min-height: 28.125rem;
```

Comment:
Since height is defined to 100 %, min-height is no longer needed?

Reply:
removed min height + width